internal/lm/nvml.go

```diff
@@ -218,6 +230,79 @@ func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	return labels, nil
 }
 
+func newImexDomainLabeler(configFile string, device []resource.Device) (Labeler, error) {
+	if configFile == "" {
+		return nil, nil
+	}
+
+	// read file and parse it
+	imexConfig, err := os.Open(configFile)
+	if err != nil {
+		klog.Warningf("failed to open imex config file: %v", err)
+		return nil, nil
+	}
+	defer imexConfig.Close()
+
```

Comment:
We need to check if the file is empty, and if so, return nil (with no warning)

Reply:
done